Packs/VaronisDataSecurityPlatform/IncidentFields/incidentfield-Varonis_Data_Security_Platform_sAMAccountName.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@vdobrotskyi-varonis Why not using the common field "sAMAccountName"?

Reply:
@tomer-pan remove custom one and changed to using common field